Mesh_3/include/CGAL/Gray_image_mesh_domain_3.h

```diff
@@ -64,12 +64,12 @@ Gray_image_mesh_domain_3
                            const Image_word_type value_outside = 0.,
                            const FT& error_bound = FT(1e-3),
                            CGAL::Random* p_rng = nullptr)
-    : Base(Wrapper(image,
+    : Base(parameters::function = Wrapper(image,
```

Comment:
This is a valid call. Except if package maintainers (@lrineau @janetournois) ask for it we can leave them.

Reply:
It is not about whether it is valid or not, but about using different parameter APIs within the same constructor call.

Reply:
I don't understand either

Reply:
same for me. It is consistent in this code isn't it?

Reply:
@MaelRL can you shed some light on what you mean here?

Reply:
In private discussion Mael mentioned that it's because `operator=` is used and he thought that the old API was deprecated, while we decided to keep it and not document it.

Reply:
I did not say I thought it was deprecated, I said I thought the decision was that the newer API should be used everywhere (even internally) as to make life easier for maintainers.